test/PlaidML/linalg-to-cpu.pp

```diff
@@ -0,0 +1,25 @@
+# linalg dialect to gpu dialect lowering pipeline
```

Comment:
we need one more .pp file for the gpu pipeline

Reply:
like this https://github.com/intel/mlir-extensions/blob/refactor/test/Integration/Dialect/Linalg/OpenCL/linalg-to-gpux-opencl ....maybe just copy that same one

Reply:
Yeah, still working on it, since some of the test cases don't run successfully yet.